study.md

```diff
@@ -40,14 +40,14 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+It means it wants to capitalize the actual string (or element) instead of making a copy of the element with the action executed.
 ```
```

Comment:
Absolutely. A good convention of terminology for this would be that it _mutates the original reference_.